File: src/containers/reto7.js
let palabras = "HOLA hola que QUE tal"
palabras = palabras.toLowerCase().split(" ")
let diccionario_frecuencias = {}

cont = 0
for (palabra in palabras){
    if (palabras[palabra] in diccionario_frecuencias){
        diccionario_frecuencias[palabras[palabra]] += 1}
    else{
        diccionario_frecuencias[palabras[palabra]] = 1}
}
for (var palabra in diccionario_frecuencias){
    frecuencia = diccionario_frecuencias[palabra]
    console.log("La palabra", palabra, "se repite", frecuencia, "veces")}

File: src/containers/reto7.py
cadena = "hola HOLA a a todos"
cadena = cadena.lower()
cadena = cadena.split(" ")
diccionario_frecuencias = {}
for palabra in cadena:
    if palabra in diccionario_frecuencias:
        diccionario_frecuencias[palabra] += 1
        print(diccionario_frecuencias[palabra])
    else:
        diccionario_frecuencias[palabra] = 1

for palabra in diccionario_frecuencias:
    frecuencia = diccionario_frecuencias[palabra]
    print(f"La palabra '{palabra}' se repite {frecuencia} veces")